modules/single-node-asg/outputs.tf

```diff
@@ -7,8 +7,3 @@ output "asg_iam_role_name" {
   value       = module.instance_profile.iam_role_name
   description = "`name` exported from the Service Data `aws_iam_role`"
 }
-
-output "data_volume_name_tag" {
-  value       = "${local.data_volume_name_prefix}-${local.az}"
-  description = "Name tag value for attached data volume"
-}
```

Comment:
Should this be kept, or why drop it?

Reply:
I am thinking a better way to do this. The value is for DLM to match the volume. Since we are having multiple now. The output does not work.